src/olympia/conf/prod/settings.py

```diff
@@ -53,6 +53,7 @@
 ES_TIMEOUT = 60
 ES_INDEXES = {k: f'{v}_{ENV}' for k, v in ES_INDEXES.items()}
 ES_DEFAULT_NUM_SHARDS = 10
+ES_COMPRESS = False
```

Comment:
TIL about this file. Is this our production settings?

Reply:
Yes